File: C++/Comparando_Numeros.cpp
#include <iostream>

using namespace std;

int main(){
	int x, y, med;
	cout << "Digite o primeiro número: ";
	cin >> x;
	cout << " \nDigite o segundo número: ";
	cin >> y;
	
 	if ( x==y ){
  		cout << "\nOs números são iguais!\n" << endl;
  	}else if ( x != y ){
  		med = (x+y)/2;
  		cout << "\nA média dos números é: " << med << endl;
 	}
 	return 0;
}

File: GO/Comparando_Numeros.go
package main

import "fmt"

func main(){
	var x int = 0
	var y int = 0
	var med int =0
	
	fmt.Print("\nDigite o primeiro número: ")
	fmt.Scanf("%d", &x);
	fmt.Print("\nDigite o segundo número: ")
	fmt.Scanf("%d", &y);
	
	if x == y{		 
    		fmt.Println("\nOs números são iguais!\n");
    	}else{
    		med = (x+y)/2
    		fmt.Println("\nA média dos números é:", med);
    	}		
}
